### DISCUSSION TITLE
Epg wont load

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#66] [Q&A] [answered]
i used Televizo for my Sony Bravia Android TV OLED 55 inches.  Am I missing something? M3u8 load just fine but no EPG like this picture below:

![7C35101C-26E0-4A70-97CD-B7731C219100](https://user-images.githubusercontent.com/16481778/187801081-4eb58d1a-26a9-471d-8a4f-0748b52e328c.jpeg)

**Comment by spotanjo3** [comment]
Help???

**Reply by BellezaEmporium** [reply, marked as answer]
The guide will be at https://iptv-org.github.io/epg/guides/us/tvtv.us.epg.xml.gz

All EPGs are compressed due to recent EPGs getting higher than 50 mbs. See iptv-org/iptv-org.github.io#230 for more info.

**Comment by dtngfan85** [comment]
See discussion thread iptv-org/iptv-org.github.io#230

**Comment by spotanjo3** [comment]
It's working thanks!
